spec/models/reading_spec.rb

```diff
@@ -41,5 +41,21 @@
       })
       expect(@reading.temp).to eq 57
     end
+
+    it "handles nil temperature properly", :vcr do
+      allow(WeatherMan).to receive(:outdoor_temp_for).and_return(nil)
+      user = create(:user, zip_code: '10216')
+      user.sensors.create(name: "abcdefgh")
+      params = ActionController::Parameters.new({
+        "reading" => {
+          "temp"=>67.42,
+          "verification"=>"c0ffee",
+          "time"=>1452241576.0,
+          "sensor_name"=>"abcdefgh"
+        }
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.